src/components/Icon/CaretIcon/examples/default.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CaretIcon from '../CaretIcon';
+
+export default React.createClass({
+	render() {
+		return (
+			<div>
+				<div>
+					<CaretIcon />
+				</div>
+				<div>
+					<CaretIcon direction="up" />
+					<CaretIcon direction="down" />
+					<CaretIcon direction="left" />
+					<CaretIcon direction="right" />
+				</div>
+				<div>
+					<CaretIcon direction="up" isBadge />
+					<CaretIcon direction="down" isBadge />
+					<CaretIcon direction="left" isBadge />
+					<CaretIcon direction="right" isBadge />
```

Comment:
IIRC we unfortunately standardized on single quotes over double quotes in the markup.